Day-II-Calculator/calculator/src/App.js

```diff
@@ -1,23 +1,102 @@
 import React from 'react';
 import './App.css';
+import CalculatorDisplay from "./components/DisplayComponents/CalculatorDisplay";
+import ActionButton from "./components/ButtonComponents/ActionButton";
+import NumberButton from "./components/ButtonComponents/NumberButton";
 
-const App = () => {
+const zero = {
```

Comment:
So you don't have to make items to pass into props, but it still works! Just extra work for you.